src/controllers/multipleCityTrips.js

```diff
@@ -0,0 +1,74 @@
+import uuidv4 from 'uuid/v4';
+import { Op } from 'sequelize';
+import datecheck from '../utils/dateCheck';
+import models from '../models';
+import response from '../utils/Response';
+
+const { serverResponse } = response;
+
+const { Trips, Locations } = models;
+/**
+ *@description A class that handles multicity travel request by a user
+ * @class MultiCityTrips
+ */
+class MultiCityTrips {
```

Comment:
Can we have the class called Trips so other trips or request related method can be put in the same file?. Your method name should still remain `multiCityRequest`.

Reply:
Dear @Stylll , my model already has its name as Trips and it is now conflicting with this? What do you advise I do?

Reply:
@sebaztyn I'm not sure how the names are conflicting. Are they conflicting in this file?. Maybe we can have a sync to look into it?

Reply:
@Styll thank you for reaching out. I made changes to resolve the conflicts within the feature.